app/assets/javascripts/backbone/views/show_edit_view.coffee

```diff
@@ -3,10 +3,6 @@ class MS.Views.ShowEditView extends Backbone.View
 
```

Comment:
We currently do not need edit links inside the modal, so changing this file is not needed.

This file is used throughout the system. Keep this file as is. This change here breaks functionality for modals throughout the system.

This is the pattern that should be used for the transaction modal, but not right at this moment. We are only implementing show transaction (not edit and delete transaction) at this time. A good example of modal usage is Add Step and Edit Step in the Timeline. Or inside the loan calendar, clicking on a step shows the modal in show view for a step. You can click edit and see the edit view. That's what type of functionality we are aiming for with this feature. But we only need the show view (and the new transaction view as well). Edit transaction is a separate feature.

A good example of show and edit views being used is Loan Details. When you first get to the page you see the show view. Then edit is shown.